corehq/apps/reports/standard/cases/utils.py

```diff
@@ -170,3 +171,8 @@ def _get_location_accessible_ids(request):
 def query_location_restricted_cases(query, request):
     accessible_ids = _get_location_accessible_ids(request)
     return query.OR(case_es.owner(accessible_ids))
+
+
+def query_location_restricted_forms(query, request):
+    accessible_ids = _get_location_accessible_ids(request)
+    return query.OR(form_es.user_id(accessible_ids))
```

Comment:
What's the difference between `query.OR(form_es.user_id(accessible_ids))` and `query.user_id(accessible_ids)`? `OR` just looks funny to me when it has only one argument

Reply:
i was following the syntax of this:
https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/reports/standard/cases/utils.py#L170-L172

Reply:
it is done correctly this way based on testing I did locally.

Reply:
It makes sense that it would work, although it's a bit nonsensical.  Here's how `query.OR` is defined:
https://github.com/dimagi/commcare-hq/blob/bfca7e35669343b457606268401d8ab9132a9ddd/corehq/apps/es/filters.py#L39-L41

That is, here's what the resulting filters for each of those two options would look:
```python
In [1]: es.filters.OR(es.forms.user_id(['ethan']))
Out[1]: {'or': [{'terms': {'form.meta.userID': ['ethan']}}]}

In [2]: es.forms.user_id(['ethan'])
Out[2]: {'terms': {'form.meta.userID': ['ethan']}}
```
They're functionally equivalent, but `query.user_id(accessible_ids)` is more to-the-point.

Looking at the commit history, I wonder whether the original author mistook `OR` to mean
```
(current queryset) OR (new condition)
```
rather than
```
(current queryset) further filtered by:
    ((new condition 1) OR (new condition 2))
```